Makefile.js

```diff
@@ -816,6 +816,20 @@ target.checkRuleFiles = function() {
             return idNewAtBeginningOfTitleRegExp.test(docText) || idOldAtEndOfTitleRegExp.test(docText);
         }
 
+        /**
+         * Check if deprecated information is in rule code and READNE.md.
+         * @returns {boolean} true if present
+         * @private
+         */
+        function hasDeprecatedInfo() {
+            const ruleCode = cat(filename);
+            const deprecatedTagRegExp = /@deprecated in ESLint/u;
+            const docText = cat(docFilename);
+            const deprecatedInfoRegExp = /This rule was .+deprecated.+in ESLint/u;
```

Comment:
Can you specify the asterisks exactly?

```suggestion
            const deprecatedInfoRegExp = /This rule was \*\*deprecated\*\* in ESLint/u;
```

Reply:
https://github.com/eslint/eslint/blob/master/docs/rules/require-jsdoc.md and https://github.com/eslint/eslint/blob/master/docs/rules/valid-jsdoc.md are exception.